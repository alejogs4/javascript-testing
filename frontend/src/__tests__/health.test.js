import React from 'react';
import { shallow } from 'enzyme';

describe('Health', () => {
  it('Test 1', () => {
    expect(true).toBe(true);
  });

  it('Enzyme', () => {
    const wrapper = shallow(<div><h1>Content</h1></div>);

    expect(wrapper).toMatchSnapshot();
  });
});
