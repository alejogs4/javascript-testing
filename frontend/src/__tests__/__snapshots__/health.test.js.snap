// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Health Enzyme 1`] = `ShallowWrapper {}`;
